meinberlin/apps/projects/templates/meinberlin_projects/includes/timeline.html

```diff
@@ -14,7 +15,7 @@
                         href="{{ project.get_absolute_url }}"
                         class="button button--light button--small timeline__button {% if obj == active_phase.module %}is-active{% endif %} {% if not object|is_offlineevent %}is-selected{% endif %}"
```

Comment:
`{% if not object|is_offlineevent %}is-selected{% endif %}` is only working as long as there is no second module.
otherwise both modules will be marked `is-selected`
currently this is no issue, but maybe we should add a comment/fixme